torchtune/data/_prompt_templates.py

```diff
@@ -6,7 +6,7 @@
 from functools import partial
 from typing import Dict, List, Protocol, Tuple, Union
 
-from torchtune.data import Message, Role
+from torchtune.data._messages import Message, Role
```

Comment:
Are these hacks for circular import issues?

Reply:
Indeed